alphastats/gui/utils/analysis.py

```diff
@@ -58,22 +46,22 @@ def show_widget(self):  # noqa: B027
         pass
 
     def do_analysis(self):
-        """Perform the analysis.
+        """Perform the analysis after an optional check for NaNs.
```

Comment:
i didn't understand what you meant by saying that this check is optional?

Reply:
will rephrase it to `Perform the analysis after some upfront method-dependent checks (e.g. or NaNs)`